package.json

```diff
@@ -111,6 +111,7 @@
   "dependencies": {
     "lodash": "^4.17.11",
     "moment-timezone": "^0.5.25",
-    "slugify": "^1.3.4"
+    "slugify": "^1.3.4",
+    "winston": "^3.2.1"
```

Comment:
@sfount suggests we pin to a specific version since we use Dependabot to manage moving from versions.